plugins/package-managers/gradle-plugin/src/main/kotlin/OrtModelBuilder.kt

```diff
@@ -207,23 +207,31 @@ internal class OrtModelBuilder : ToolingModelBuilder {
                                 selectedComponent.dependencies.toOrtDependencies(poms)
                             }
 
+                            if (modelBuildingResult == null) {
+                                val message = "POM is missing for $id"
+                                logger.warn(message)
+                                warnings += message
+                            }
+
```

Comment:
Let's move this directly after line 201 to handle the `null` case where may occur.

Reply:
Come to think of it, I suppose it should also check if the dependency ID is not in `visitedDependencies` to avoid having multiple warnings?

Reply:
Yeah, extending the check to `if (modelBuildingResult == null && id !in visitedDependencies)` sounds like a good idea.

Reply:
Thanks for your contribution!